src/vsphere_cpi/lib/cloud/vsphere/cloud.rb

```diff
@@ -662,6 +662,50 @@ def info
       {'stemcell_formats' =>  ['vsphere-ovf', 'vsphere-ova']}
     end
 
+    #creates T1 router and virtual switch attached to it
+    def create_subnet(subnet_definition)
```

Comment:
methods should be called create_network and delete_network

Reply:
Was thinking about it. I understand that we are creating network infrastructure, but on director side you asking to create a subnet and you give a subnet definition. 
As  I remember your point for calling this `*_network` was that we are passing `networks_spec` to VM. 
Would it be more confusing if we call `create_network` on a BOSH side and give it a `subnet_definition` and it returns us `cloud_properties` for a subnet?

Reply:
@cppforlife ^^